### PR TITLE
Forums Fixes

### PR DESCRIPTION
**Pull request by clintonb**
- Updating bundler
- Installing bundle as forums service user

OPS-1243

Fix for issues introduced by #2666.

### CODE REVIEW

**Comment by clintonb** [comment]
FYI @e0d @joxper @jzoldak @macdiesel @fredsmith 

Sandbox build (with only forums): http://jenkins.edx.org:8080/view/Ansible/job/ansible-provision/4821/console

File: playbooks/roles/forum/tasks/deploy.yml [hunk 1]
```diff
@@ -49,12 +49,17 @@
     - install
     - install:code
 
-# TODO: This is done as the common_web_user
-# since the process owner needs write access
-# to the rbenv
+- name: update bundler
+  shell: gem update bundler
```
Comment (macdiesel) [inline playbooks/roles/forum/tasks/deploy.yml]: We should pin this to a version.

**Comment by e0d** [comment]
I like this in many ways, but am curious about the history of the original change and the missing context for the comment you remove.  All that being said, not allowing the web runtime user to write to this directory is a very good change.

**Comment by joxper** [comment]
Hello @clintonb Now the problem is in the rbenv, I am not very sure about this:

```
TASK: [rbenv | fail rbenv_user required for role] ***************************** 
skipping: [localhost]

TASK: [rbenv | fail rbenv_dir required for role] ****************************** 
skipping: [localhost]

TASK: [rbenv | fail rbenv_ruby_version required for role] ********************* 
skipping: [localhost]

TASK: [rbenv | create rbenv user {{ forum_user }}] **************************** 
ok: [localhost]

TASK: [rbenv | create rbenv dir if it does not exist] ************************* 
ok: [localhost]

TASK: [rbenv | install build depends] ***************************************** 
ok: [localhost] => (item=curl,build-essential,libcurl4-openssl-dev,libreadline-dev,libssl-dev,libxml2-dev,libxslt1-dev,zlib1g-dev)

TASK: [rbenv | update rbenv repo] ********************************************* 
failed: [localhost] => {"failed": true}
msg: Failed to download remote objects and refs:  error: cannot open .git/FETCH_HEAD: Permission denied

FATAL: all hosts have already failed -- aborting
```

**Comment by clintonb** [comment]
@joxper I am still working on the fix. I'll let you know once I have a successful build.

**Comment by joxper** [comment]
@clintonb I erase everything in /edx and start from scratch, it went successful (the commits changes) the only remaining issue for me and I think for others is this issue https://github.com/edx/configuration/issues/2668, it was happening before your commit and now is happening again, please can you take a look at this.

**Comment by clintonb** [comment]
Build with working forum service (included a hack to build the forum service first): http://jenkins.edx.org:8080/view/Ansible/job/ansible-provision/4855/console

Build without hack (may fail due to unrelated edxapp database migration issue): http://jenkins.edx.org:8080/view/Ansible/job/ansible-provision/4857/console

**Comment by clintonb** [comment]
Please review @e0d @fredsmith @macdiesel

**Comment by clintonb** [comment]
FYI @bderusha

**Comment by e0d** [comment]
:+1:

**Comment by regisb** [comment]
Any chance this could be backported to a 'cypress' branch? Or do we have to checkout the master branch to install cypress?

**Comment by clintonb** [comment]
@regisb I have no plans to backport. (I honestly had no plans for this fix!) Most of our resources are focused on deploying Dogwood (which might have this change). I'd recommend migrating. However, if that is not feasible, I can review your pull request if you open one against the Cypress branch.

**Pull request merged**